loyalty_partner_applicability/models/loyalty_program.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, models
+
+
+class LoyaltyProgram(models.Model):
+    _inherit = "loyalty.program"
+
+    @api.model
+    def _program_type_default_values(self):
+        res = super()._program_type_default_values()
+        program_types_to_update = [
+            "promotion",
+            "loyalty",
+            "promo_code",
+            "buy_x_get_y",
+            "next_order_coupons",
+        ]
+        for program_type in program_types_to_update:
+            res[program_type]["rule_ids"][1][2].update({"rule_partners_domain": "[]"})
```

Comment:
What if another module added a different structure?

Reply:
Possibility considered, thank you